.github/workflows/e2e-compatibility.yaml

```diff
@@ -22,7 +28,8 @@ env:
   REGISTRY: ghcr.io
   ORG: cosmos
   IMAGE_NAME: ibc-go-simd
-  RELEASE_BRANCH: '${{ inputs.release-branch }}'
+  RELEASE_BRANCH: ${{ inputs.release-branch || 'main' }}
+  IBC_GO_VERSION: ${{ inputs.ibc-go-version || 'main-cron-job' }}
```

Comment:
Wonder if there's anything that's gonna cry over this? could just use `latest` if it comes to whinge